test/distributed/test_c10d_nccl.py

```diff
@@ -4385,7 +4385,7 @@ def started_or_scheduled(self, timing_enabled):
 class NCCLTraceTest(NCCLTraceTestBase):
     def _verify_trace(self, t, include_collectives, timing_enabled, is_json):
         ver = t["version"]
-        self.assertEqual(ver, "2.5")
+        self.assertEqual(ver, "2.7")
```

Comment:
why didn't this error when the version was incorrectly (?) 2.6 above?

Reply:
oh it is a rebase slowness. The base should be 2.6...